apps/web/pages/_app.tsx

```diff
@@ -1,15 +1,22 @@
 import type { IncomingMessage } from "http";
+import type { NextApiRequest } from "next";
 import type { AppContextType } from "next/dist/shared/lib/utils";
 import React from "react";
 
 import { trpc } from "@calcom/trpc/react";
 
 import type { AppProps } from "@lib/app-providers";
+import { useRedirectIfPlatformUser } from "@lib/useRedirectIfPlatformUser";
 
 import "../styles/globals.css";
 
 function MyApp(props: AppProps) {
   const { Component, pageProps } = props;
+  const isPlatformUser = pageProps.isPlatformUser;
+
+  // custom hook to handle redirection for platform users
```

Comment:
For future reference in case someone wants to modify `_app.tsx` and doesn't know what this hook does they don't need to go inside the hook, the comments pretty much explain what the hook does